pkgs/development/tools/rust/cargo-shuttle/default.nix

```diff
@@ -30,13 +28,10 @@ rustPlatform.buildRustPackage rec {
   };
 
   nativeBuildInputs = [
-    curl
     pkg-config
   ];
 
   buildInputs = [
-    curl
-    libgit2_1_5
     openssl
```

Comment:
According to [this commit](https://github.com/shuttle-hq/shuttle/pull/879), it might be possible to remove `openssl`.